omnigibson/robots/tiago.py

```diff
@@ -1,6 +1,6 @@
 import os
-
 import numpy as np
+from telemoma.human_interface.teleop_core import TeleopAction
```

Comment:
Don't import this here. Creates an unnecessary dependency. Remove the type hint it's used for.